src/app.js

```diff
@@ -1,6 +1,16 @@
 // A sample app that listens to messages posted to a space in IBM
```

Comment:
Many many changes here probably caused by your editor. Can you please re-apply the minimum set of changes that will do what you intended to do? and I'll be happy to review again :) Thanks!